test/Microsoft.AspNet.Mvc.TestCommon/TestModelMetadataProvider.cs

```diff
@@ -63,7 +62,7 @@ public IMetadataBuilder ForProperty(Type containerType, string propertyName)
             var property = containerType.GetRuntimeProperty(propertyName);
             Assert.NotNull(property);
 
-            var key = ModelMetadataIdentity.ForProperty(property.PropertyType, propertyName, containerType);
+            var key = ModelMetadataIdentity.ForProperty(property.PropertyType, property.Name, containerType);
```

Comment:
curious: when would the `propertyName` value be different from `property.Name`?
and why ditch the parameter in favour of the property?